docs/getting-started/getting-started-via-code.md

```diff
@@ -88,7 +88,7 @@ vdoctl configure compatibility-matrix
 ✔ Web URL
 Web URL https://raw.githubusercontent.com/asifdxtreme/Docs/master/sample/matrix/matrix.yaml
 ```
-Note: You can either use this sample url or create your own matrix.
+You can either use this sample url or create your own matrix.
```

Comment:
Will this current Makefile also adds the matrix, if yes then we can skip this step